tests/wdio/utilities-spec.js

```diff
@@ -0,0 +1,70 @@
+Terra.describeViewports('utilities', ['huge'], () => {
+  it('checks accessibility', () => {
+    browser.url('/single-page-test');
+    browser.click('[class*="UtilityMenuHeaderButton-module"]');
+    Terra.validates.element({ selector: '#root' });
+  });
+
+  it('selects config', () => {
+    browser.url('/single-page-test');
+    browser.click('[class*="UtilityMenuHeaderButton-module"]');
+    browser.click('[class*="PopupMenuListItem-module__item"]:nth-child(1)');
+    Terra.validates.element('selects config', { selector: '#root' });
+  });
+
+  it('changes config', () => {
+    browser.click('#terra-dev-site-locale-select');
+    browser.click('#terra-select-option-en-AU');
+
+    browser.click('#terra-dev-site-theme-select');
+    browser.click('[id="terra-select-option-Terra Dev Site Test Theme"]');
+
+    browser.click('#terra-dev-site-direction-select');
+    browser.click('#terra-select-option-rtl');
+    Terra.validates.element('changes config', { selector: '#root' });
+
+    browser.click('#submit');
+    Terra.validates.element('config changed', { selector: '#root' });
+
+    browser.click('[class*="UtilityMenuHeaderButton-module"]');
+    browser.click('[class*="PopupMenuListItem-module__item"]:nth-child(1)');
```

Comment:
Are the selectors used throughout this test specific to terra-dev-site? If not, we could have to update these if these classes ever change

Reply:
Yes, we may have to update these if the components change.